components/script/dom/oscillatornode.rs

```diff
@@ -13,23 +13,25 @@ use crate::dom::bindings::codegen::Bindings::OscillatorNodeBinding::OscillatorNo
 use crate::dom::bindings::codegen::Bindings::OscillatorNodeBinding::{
     self, OscillatorOptions, OscillatorType,
 };
-use crate::dom::bindings::error::Fallible;
+use crate::dom::bindings::error::{Error, ErrorResult, Fallible};
 use crate::dom::bindings::reflector::reflect_dom_object;
 use crate::dom::bindings::root::{Dom, DomRoot};
 use crate::dom::window::Window;
 use dom_struct::dom_struct;
-use servo_media::audio::node::AudioNodeInit;
+use servo_media::audio::node::{AudioNodeInit, AudioNodeMessage};
+use servo_media::audio::oscillator_node::OscillatorNodeMessage;
```

Comment:
```
error[E0432]: unresolved import `servo_media::audio::oscillator_node::OscillatorNodeMessage`
  --> components/script/dom/oscillatornode.rs:22:5
   |
22 | use servo_media::audio::oscillator_node::OscillatorNodeMessage;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^---------------------
   |     |                                    |
   |     |                                    help: a similar name exists in the module: `OscillatorNode`
   |     no `OscillatorNodeMessage` in `audio::oscillator_node`
```

You still need to update `servo-media` to get the changes where you introduced `OscillatorNodeMessage`.

Run:
```
./mach cargo-update -p servo-media
```

And commit the changes done to the `Cargo.lock` file.

Reply:
Woops, I had done that locally but didn't know I needed to commit the changes to Cargo.lock. Now that I've done that, it says this branch has merge conflicts that need to be resolved.